src/pages/JoinModal/JoinModal.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import './JoinModal.scss';
+
+function JoinModal() {
+  return (
+    <div className="zarnaraCreateContainer">
```

Comment:
컴포넌트의 최상위 태그에는 컴포넌트명과 동일한 className을 camelCase로 부여해주세요!
```suggestion
    <div className="joinModal">
```

Reply:
그리고 zaranara인데 zarnara 🙈